study/cyber-tutor/quiz/js/app.js

```diff
@@ -0,0 +1,97 @@
+const myQuestions = [
+  {
+    question: "What does cybersecurity protect?",
+    answers: {
```

Comment:
I like this system of validating if answers are correct or not, but I think that having the correctAnswer property be the answer content instead of the letter would allow you to shuffle multiple choice questions up making it more varied. This would stop people from mindlessly taking the quiz until they get the right answers.